### PR TITLE
Adding Rstudio 4.1.1

### LINKED ISSUE #243: [SDKTECHNO-99] Doc RStudio to update
Missing references to new rstudioadmin env var

### PR DESCRIPTION
**Pull request by TFellet**
Closes #243, #258, #155

### CODE REVIEW

File: technologies/app/rstudio-4.1/rstudio-4.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,101 @@
+FROM rocker/tidyverse:4.1.1
+
+LABEL maintainer="Saagie <tony@saagie.com>"
+
+USER root
+
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV CDH_MAIN_VERSION 5
+ENV CDH_VERSION ${CDH_MAIN_VERSION}.7.1
```
Comment (skydjol) [inline technologies/app/rstudio-4.1/rstudio-4.1/Dockerfile:10]: On rstudio 3.6.2 
```suggestion
ENV CDH_VERSION ${CDH_MAIN_VERSION}.13.0
```

File: technologies/app/rstudio-4.1/rstudio-4.1/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+ENV HADOOP_VERSION 2.7.0
```
Comment (skydjol) [inline technologies/app/rstudio-4.1/rstudio-4.1/Dockerfile:11]: Hadoop 2.7 and on rstudio 3.6.2, it's hadoop 2.6?

**Review by skydjol — DISMISSED** [review]

**Review by fabienmifsud — APPROVED** [review]

File: technologies/app/rstudio-4.1/rstudio-4.1/Dockerfile (hunk 2 above)
Comment (TFellet) [reply technologies/app/rstudio-4.1/rstudio-4.1/Dockerfile:11]: hadoop command line 2.7 adds recursive ls, it's nice but not mandatory, I can change it to 2.6 if necessary

File: technologies/app/rstudio-4.1/rstudio-4.1/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,101 @@
+FROM rocker/tidyverse:4.1.1
```
Comment (tquiviger) [inline technologies/app/rstudio-4.1/rstudio-4.1/Dockerfile:1]: Why not having a FROM Saagie R image (most of the  content of the Dockerfile between R and Rstudio are identical) and adding nginw and the correct entrypoint for Rstudio?

**Review by tquiviger — APPROVED** [review]

**Pull request merged**